frontend/src/control/fader.rs

```diff
@@ -142,6 +144,16 @@ impl Component for Fader {
                 self.mouse_mode = MouseMode::Normal;
                 true
             }
+            FaderMsg::Scroll(dir) => {
+                let delta = match dir {
+                    Scroll::Up(delta) => delta,
+                    Scroll::Down(delta) => delta * -1.0,
+                };
+                let factor = 0.0001;
+                let fader_value = util::clamp(0.0, 1.0, self.props.value + delta * factor);
```

Comment:
`util::clamp` is nice, but I think it would be better if there was a way to have the `onchange` callback do the clamping. Otherwise it's up to the message sender to clamp and there's an easy opening for a bug there if the clamping is forgotten. If I had forgotten this line, for instance, a rapid scroll could possibly blow out the speakers (the device may still clamp at some point, but still...)

Not sure what this looks like. `self.update_value(value)` which wrapped the emit could work, but still leaves open the possibility of directly emitting a bad value. Still an improvement, though.